build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1312,6 +1323,57 @@ message OutputSymlink {
   NodeProperties node_properties = 4;
 }
 
+// The termination result of a process, as reported by the operating
+// system to the parent process (e.g., using wait*() on POSIX-like
+// systems, or GetExitCodeProcess() on Microsoft Windows).
```

Comment:
I'd suggest removing or weakening the statement "as reported by the operating system..." . This structure of exit_code + process_termination_result I think would actually extend to other concepts as well in the future.

For example, a pattern I expect to need at some point is "flake due to intermediate on-worker infrastructure", where there is a logical test that we care about the result of, but also some intermediate scaffolding that's been injected to run it where we'd like to be able to distinguish flakes due to the scaffolding itself (that doesn't necessarily imply a problem with the test) from flakes of the test.

(To be clear, don't want to extend your PR to cover this; that can be a future discussion. I'd just say that the "process termination result" need not be _only_ OS-provided information.